packages/tracking/src/hooks/use_track.ts

```diff
@@ -56,6 +55,7 @@ export const useTrack = (): TrackFunction => {
 
     log('tracking to dataLayer', event, {
       section: map?.section,
+      page_name: map?.page_name,
```

Comment:
`page_name` only in log message but not in real dataLayer push

Reply:
tacking data now used for log and push 😅